### DISCUSSION TITLE
Welcome to Geo-SAM Discussions!

### DISCUSSION THREAD

**Discussion by coolzhao** [coolzhao/Geo-SAM#28] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by Martin-Dawson** [comment]
Has there been any consideration of adding fining tuning capabilities as per the following discussion: https://encord.com/blog/learn-how-to-fine-tune-the-segment-anything-model-sam/
